manifests/params.pp

```diff
@@ -0,0 +1,16 @@
+# Main class
+class cerebro::params {
+  $version        = '0.6.5'
+  $service_ensure = 'running'
+  $service_enable = true
+  $secret         = undef #Please specify a key
+  $hosts          = undef
+  $basepath       = '/'
+  $cerebro_user   = 'cerebro'
+  $package_url    = "https://github.com/lmenezes/cerebro/releases/download/v${version}/cerebro-${version}.zip"
```

Comment:
I don't think this will work.
eg
```puppet
class { 'cerebro':
  version => '0.6.6',
}
```
and version 0.6.5 will still be downloaded.